runtime/kubernetes-1ppc/init-st2.yml

```diff
@@ -10,7 +10,7 @@ spec:
     spec:
       containers:
       - name: init-st2-default-files
-        image: stackstorm/stackstorm-1ppc:latest
+        image: stackstorm/stackstorm:latest
```

Comment:
I believe this is not possible with plain manifest file, and instead it should be achieved by providing a Helm chart for st2.